Resources/config/custom.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <!-- you should define all services from fos_oauth_server.service configuration section explicitly -->
```

Comment:
Since we're adding this empy file, maybe add a (commented-out) sample configuration of required services? We could at least enumerate them here.

Reply:
don't create this file at all (and handle this when loading the file in the DI extension)

Reply:
We would have to add if condition to cover the special case, I am in favor of keeping the file and adding the example

Reply:
What should I do?

Reply:
Ok, I'll do